Source/FilePreviewPanel.h

```
@@ -7,13 +7,17 @@
 #include <vector>
 #include <string>
 #include <functional>
+#include <future>
+#include <atomic>
+#include <vector>
```

Clang (14.0.6)

[warning] duplicate include

(readability-duplicate-include)